src/index.js

```diff
@@ -81,4 +81,40 @@ function RebillyExperimentalAPI({apiKey = null, sandbox = false, timeout = baseT
     return createExperimentalApiInstance({apiHandler});
 }
 
-export {Errors as RebillyErrors, RebillyExperimentalAPI, cancellation};
+/**
+ * Create an instance of the experimental Rebilly API
+ * @param apiKey {string} publishable API key; if provided will be used for all requests
```

Comment:
how will you use a JWT?

Reply:
JWT is an option below

Reply:
Does it seem that the JWT should be used in priority over the publishable key?

Reply:
Normally the JWT is set with `setSessionToken` https://github.com/Rebilly/rebilly-js-sdk/blob/9e2b2ddae1e39752e2187baf55cf032e91bba7fc/src/create-api-handler.js#L119
So it's more explicitly preferring the JWT in that case.

From looking at the code if both authorization methods are provided then the apiKey is used.

Reply:
In fact the JWT is not settable in the configuration. A JWT is only useable if provided with the `setSessionToken` function.

It's not a configuration option, just passed here for creation for if the setSesionToken is called in the future.